### PR TITLE
add t.appendChild helper

### PR DESCRIPTION
**Pull request by tony-go**
According to https://github.com/hyperdivision/vhs-tape/issues/19

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -87,6 +87,12 @@ function create (delay, fn) {
           el.dispatchEvent(new window.KeyboardEvent(event || 'keydown', { key: c }))
         }
         return t.delay()
+      },
+      async appendChild (el) {
+        if (!el) return
+        t.element.appendChild(el)
+        await t.onload(el)
+        return t.delay()
```
Comment (bcomnes) [inline index.js]:
```suggestion
```

File: index.js [hunk 2]
```diff
@@ -87,6 +87,12 @@ function create (delay, fn) {
           el.dispatchEvent(new window.KeyboardEvent(event || 'keydown', { key: c }))
         }
         return t.delay()
+      },
+      async appendChild (el) {
+        if (!el) return
+        t.element.appendChild(el)
+        await t.onload(el)
```
Comment (bcomnes) [inline index.js]:
```suggestion
        return t.onload(el)
```

File: index.js (hunk 1 above)
Comment (bcomnes) [reply index.js]: Lets not have a delay hook here.

**Review by bcomnes — CHANGES_REQUESTED** [review]
Looks good except for the delay hook.

File: index.js (hunk 1 above)
Comment (tony-go) [reply index.js]: Thanks :) I didn't see that onload hook had his own delay hook.

**Comment by tony-go** [comment]
I update readme according to https://github.com/hyperdivision/vhs-tape/issues/17

**Review by bcomnes — APPROVED** [review]
Amazing ✌️

**Pull request merged**